SparkPost/2web2ui
# FE-1175: Remove "allow_dashboard_v2" Feature Flag

FE-1175: Remove "allow_dashboard_v2" Feature Flag

### What Changed

- For Hibana only, remove the "allow_dashboard_v2" requirement for the rendering of "/dashboardV2"
- For Hibana users, render the new Dashboard page at the "/dashboard" route
- For OG users, render the original Dashboard page
- Removed feature flag - allow_dashboard_v2
- Removed automated tests for the feature flag

### How To Test

- in OG theme /dashboard routes to old dashboard page
- in Hibana theme /dashboard routes to new dashboard page
- Check if this change works correctly for different roles - 
    - Admin, Developer, Templates - landing page for these role should be old dashboard in OG theme and new dashboard in hibana theme
    - Reporting - landing page for this role should be Summary Report in OG theme and new dashboard in hibana theme
    - Subaccount Reporting - landing page should be Summary Report in OG theme and Hibana. This role shouldn't have access to dashboard in any theme.

### To Do
